src/common/lib/client/realtime.ts

```diff
@@ -161,6 +161,15 @@ class Channels extends EventEmitter {
     return channel;
   }
 
+  getDerived(name: string, deriveOptions: API.Types.DeriveOptions, channelOptions?: ChannelOptions) {
+    if (deriveOptions.filter) {
+      const filter = encodeURIComponent(deriveOptions.filter);
```

Comment:
We are making a change so that we now use a base64 encoding instead of URL encoding. See this PR: https://github.com/ably/realtime/pull/4835/files

Can we please update this PR to reflect these changes before we merge?